restcomm/restcomm.mgcp/src/main/java/org/restcomm/connect/mgcp/AsrSignal.java

```diff
@@ -57,6 +60,9 @@ public AsrSignal(String driver, String lang, List<URI> initialPrompts, String en
         this.timeAfterSpeech = timeAfterSpeech;
         this.hotWords = hotWords;
         this.lang = lang;
+        this.input = input;
+        this.minNumber = numberOfDigits;
+        this.maxNumber = numberOfDigits;
```

Comment:
can we add a code comment here why we are assigning both members same incoming value